impoved send_message func from broadcaster module

itisnotyourenv opened the pull request:
I added the possibility of sending messages with InlineKeyboardMarkup to function send_message from the broadcaster module.

itisnotyourenv committed:
impove send_message func from broadcaster module

Latand reviewed (changes requested):
I don't see you changed the actual broadcast function

itisnotyourenv commented:
> I don't see you changed the actual broadcast function

My changes were for a way when we need to send messages to other users from handlers.
I got your review and updated code.

```
@router.callback_query()
async def buy_product(call: types.CallbackQuery, bot: Bot, translator: MultilingualBot, extra_button: ExtraButton):
    product = call.data.split("_")[1]

    response, ok = await buy_product_request(user_id=call.from_user.id, product_id=product_id)
    if not ok:
        return await call.answer(item, show_alert=True)
   
    # Now I can send notification with markup 
    await send_message(bot=bot, user_id=response.product_owner_id, text="Your sold 1 item", reply_markup=markup)
```

itisnotyourenv committed:
update after review

Latand commented:
There is no such need in using this safe send message method in handlers, actually
But might be good for broadcasting
